common/graphicalculturetypes/wc_graphicalculturetype.txt

```diff
@@ -493,4 +493,13 @@ nightelffelgfx = {
 }
 highbornefelgfx = {
 	skin_tone = 0
+}
+nightelfundeadgfx = {
+	skin_tone = 0
+}
+highborneundeadgfx = {
+	skin_tone = 0
+}
+highelflighttouchedgfx = {
```

Comment:
To make race traits work correct you need to add it in `has_highelf_gfx_trigger`.